tools/sandbox/application.py

```diff
@@ -175,7 +175,8 @@ def analyze(self) -> None:
                     )
                     LOG.debug("\n".join(model_verification_errors))
                 LOG.debug(line)
-                emit("pysa_results_channel", {"type": "output", "line": line})
+                if "INFO" in line or "ERROR" in line:
```

Comment:
Hey a simple suggestion would be to merge this with the previous ` if` statements. That would give us the opportunity to make the code a bit tidier. This approach works though - just a suggestion :)

Reply:
made the changes 😄